modules/virt-about-reclaiming-persistent-volumes.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-about-reclaiming-persistent-volumes_{context}"]
+
+= About reclaiming persistent volumes
+
+When you reclaim a PV, you unbind the PV from a persistent volume claim (PVC)
```

Comment:
s/PV/persistent volume (PV)
You need to explain the initialism on first use in the body text.

Reply:
Done